content/doc/book/security/scripted.adoc

```diff
@@ -0,0 +1,70 @@
+---
+title: Scripted Security
+layout: section
+---
+ifdef::backend-html5[]
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+Apache Groovy scripts can be executed on the Jenkins controller and agents through various mechanisms including:
+
+* link:/doc/book/managing/script-console/[Script Console]
+* link:https://plugins.jenkins.io/groovy/[Groovy plugin] 
+when using the "Execute system Groovy script" step
+* link:https://plugins.jenkins.io/email-ext/[Extended Email plugin]
+* link:https://plugins.jenkins.io/groovy/[Job DSL plugin]
+* link:https://www.jenkins.io/doc/book/pipeline/[Pipelines]
+
+Scripts are powerful and useful tools but they must be managed carefully
+to ensure that they are not used to compromise your installation:
+
+* Limit the people to whom the the "Overall/Runscripts" authorization is granted.
+This allows people to use the
+link:/doc/book/managing/script-console/[Script Console].
+This authorization allows one to type in and execute an Apache Groovy script
+in a largely unrestricted environment
+so it should only be available to highly trusted, sophisticated users.
+
+* *NEVER* run Jenkins or agents as the root or system administrator user on the operating system
+because this could allow processes to bypass normal protections for the Script Console.
+
+Two facilities are provided that  allow users with lesser permissions
+(such as "Job/Configure" or "Job/Run") to run scripts that may be associated with plugins or their applications:
+
+* Jenkins maintains a list of "approved" scripts that can not do any damage.
+If a user attempts to run a script that is not approved,
+the script is blocked until an administrator reviews the script and approves it
+using the _Manage Jenkins >> In-process Script Approval_ screen.
+See the link:/doc/book/managing/script-approval/[In-process Script Approval] documentation for more information.
+In most cases, you should use the
+link:http://localhost:4242/doc/book/managing/script-approval/#approve-assuming-permissions-check[Approve assuming permissions check] option rather than the simple Approve option.
```

Comment:
Just no.

```suggestion
```

Anyway this (very dubious) feature applies to the sandbox, _not_ whole-script approval, so it is not even in the right bullet point.